### PR TITLE
Move TODOs over to GitHub issue numbers

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #37*

*Description of changes:*
Added GitHub issue numbers for TODOs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/elasticodbc/es_types.h [hunk 1]
```diff
@@ -295,7 +295,6 @@ typedef struct ColumnInfo {
     }
 } ColumnInfo;
 
-// TODO: AE-111 determine if there's a better way to hold the server_info -
 // perhaps a vector of smart pointers?
```
Comment (jordanw-bq) [inline src/elasticodbc/es_types.h]: Should be removed as part of previous line

File: src/elasticodbc/es_parse_result.cpp [hunk 2]
```diff
@@ -331,7 +331,7 @@ bool AssignRowData(const json_arr_it &row, const schema_type &row_schema,
     size_t i = 0;
     for (auto row_column = row.value_begin(); i < row_schema.size();
          ++row_column, ++i) {
-        // TODO: Milestone-3 use row_schema[i].second's type (or something else)
+        // Use row_schema[i].second's type (or something else)
```
Comment (jordanw-bq) [inline src/elasticodbc/es_parse_result.cpp]: Either remove the whole comment, or make a new issue for it and keep the TODO

File: src/elasticodbc/es_types.h (hunk 1 above)
Comment (rupal-bq) [reply src/elasticodbc/es_types.h]: removed

File: src/elasticodbc/es_parse_result.cpp (hunk 2 above)
Comment (rupal-bq) [reply src/elasticodbc/es_parse_result.cpp]: removed

File: src/elasticodbc/es_info.cpp [hunk 3]
```diff
@@ -83,7 +83,7 @@ const std::unordered_map< int, std::vector< int > > sql_es_type_map = {
       ES_TYPE_UNSUPPORTED}},
     {SQL_TYPE_TIMESTAMP, {ES_TYPE_DATETIME}}};
 
-// TODO AE-152: Refactor this to improve performance
+// TODO #325 (SQL Plugin)- Refactor this to improve performance
```
Comment (lyndonb-bq) [inline src/elasticodbc/es_info.cpp]: I think this was referring to refactoring the way the polymorphism is setup and function calls are performed, as is independent of the SQL Plugin. I don't think this is a performance bottleneck, so perhaps it's not required, but I don't think 325 is relevant to this.

File: src/elasticodbc/es_info.cpp [hunk 4]
```diff
@@ -661,7 +661,7 @@ ESAPI_Tables(HSTMT hstmt, const SQLCHAR *catalog_name_sql,
         //  arguments
         TableResultSet result_type = TableResultSet::All;
 
-        // TODO: Refactor logic
+        // TODO #324 (SQL Plugin)- Refactor logic
```
Comment (lyndonb-bq) [inline src/elasticodbc/es_info.cpp]: I think this ticket was referring to the logic of the if else statements here, not related to SQL plugin

File: src/elasticodbc/es_info.cpp (hunk 3 above)
Comment (rupal-bq) [reply src/elasticodbc/es_info.cpp]: removed

File: src/elasticodbc/es_info.cpp (hunk 4 above)
Comment (rupal-bq) [reply src/elasticodbc/es_info.cpp]: removed

**Pull request merged**